oxide/crates/node/.gitignore

```diff
@@ -121,7 +121,7 @@ dist
 .AppleDouble
 .LSOverride
 
-# Icon must end with two 

```

Comment:
There was a Windows line end (CR) here.

Reply:
Yea, that one is on purpose. Though in the comment you could just say `\r`. Line 125 `Icon` should end with two windows line endings as well. You can see more about the file here: https://superuser.com/questions/298785/icon-file-on-os-x-desktop

Reply:
(two are required because of a quirk of gitignore — even though the filename is spelled `Icon\r`)

Reply:
Thank you @thecrypticace!
Added it name: Icon must end with two CR characters
https://en.wikipedia.org/wiki/ASCII#End_of_line